clang/test/CodeGenSYCL/intel-fpga-local.cpp

```diff
@@ -1,49 +1,53 @@
 
-//CHECK: [[ANN1:@.str]] = {{.*}}{memory:DEFAULT}{sizeinfo:4}{numbanks:4}
-//CHECK: [[ANN2:@.str.[0-9]*]] = {{.*}}{register:1}
-//CHECK: [[ANN3:@.str.[0-9]*]] = {{.*}}{memory:DEFAULT}{sizeinfo:4}
-//CHECK: [[ANN4:@.str.[0-9]*]] = {{.*}}{memory:DEFAULT}{sizeinfo:4}{bankwidth:4}
-//CHECK: [[ANN5:@.str.[0-9]*]] = {{.*}}{memory:DEFAULT}{sizeinfo:4}{max_private_copies:8}
-//CHECK: [[ANN10:@.str.[0-9]*]] = {{.*}}{memory:DEFAULT}{sizeinfo:4}{pump:1}
-//CHECK: [[ANN11:@.str.[0-9]*]] = {{.*}}{memory:DEFAULT}{sizeinfo:4}{pump:2}
-//CHECK: [[ANN12:@.str.[0-9]*]] = {{.*}}{memory:DEFAULT}{sizeinfo:4}{merge:foo:depth}
-//CHECK: [[ANN13:@.str.[0-9]*]] = {{.*}}{memory:DEFAULT}{sizeinfo:4}{merge:bar:width}
-//CHECK: [[ANN14:@.str.[0-9]*]] = {{.*}}{max_replicates:2}
-//CHECK: [[ANN15:@.str.[0-9]*]] = {{.*}}{memory:DEFAULT}{sizeinfo:4}{simple_dual_port:1}
-//CHECK: [[ANN6:@.str.[0-9]*]] = {{.*}}{memory:BLOCK_RAM}{sizeinfo:4}
-//CHECK: [[ANN7:@.str.[0-9]*]] = {{.*}}{memory:MLAB}{sizeinfo:4}
-//CHECK: [[ANN8:@.str.[0-9]*]] = {{.*}}{memory:DEFAULT}{sizeinfo:4}{bankwidth:8}
-//CHECK: [[ANN9:@.str.[0-9]*]] = {{.*}}{memory:DEFAULT}{sizeinfo:4}{max_private_copies:4}
-//CHECK: [[ANN16:@.str.[0-9]*]] = {{.*}}foobar
-//CHECK: [[ANN17:@.str.[0-9]*]] = {{.*}}{memory:MLAB}{sizeinfo:4,500}
-//CHECK: [[ANN18:@.str.[0-9]*]] = {{.*}}{memory:BLOCK_RAM}{sizeinfo:4,10,2}
-//CHECK: [[ANN19:@.str.[0-9]*]] = {{.*}}{memory:DEFAULT}{sizeinfo:4}{numbanks:4}{bank_bits:4,5}
-//CHECK: [[ANN20:@.str.[0-9]*]] = {{.*}}{memory:DEFAULT}{sizeinfo:4,10,2}{bankwidth:16}{numbanks:2}{bank_bits:0}
-//CHECK: [[ANN21:@.str.[0-9]*]] = {{.*}}{memory:MLAB}{sizeinfo:4}{numbanks:8}{bank_bits:5,4,3}
+//CHECK-DEVICE: [[ANN1:@.str]] = {{.*}}{memory:DEFAULT}{sizeinfo:4}{numbanks:4}
+//CHECK-DEVICE: [[ANN2:@.str.[0-9]*]] = {{.*}}{register:1}
+//CHECK-DEVICE: [[ANN3:@.str.[0-9]*]] = {{.*}}{memory:DEFAULT}{sizeinfo:4}
+//CHECK-DEVICE: [[ANN4:@.str.[0-9]*]] = {{.*}}{memory:DEFAULT}{sizeinfo:4}{bankwidth:4}
+//CHECK-DEVICE: [[ANN5:@.str.[0-9]*]] = {{.*}}{memory:DEFAULT}{sizeinfo:4}{max_private_copies:8}
+//CHECK-DEVICE: [[ANN10:@.str.[0-9]*]] = {{.*}}{memory:DEFAULT}{sizeinfo:4}{pump:1}
+//CHECK-DEVICE: [[ANN11:@.str.[0-9]*]] = {{.*}}{memory:DEFAULT}{sizeinfo:4}{pump:2}
+//CHECK-DEVICE: [[ANN12:@.str.[0-9]*]] = {{.*}}{memory:DEFAULT}{sizeinfo:4}{merge:foo:depth}
+//CHECK-DEVICE: [[ANN13:@.str.[0-9]*]] = {{.*}}{memory:DEFAULT}{sizeinfo:4}{merge:bar:width}
+//CHECK-DEVICE: [[ANN14:@.str.[0-9]*]] = {{.*}}{max_replicates:2}
+//CHECK-DEVICE: [[ANN15:@.str.[0-9]*]] = {{.*}}{memory:DEFAULT}{sizeinfo:4}{simple_dual_port:1}
+//CHECK-DEVICE: [[ANN6:@.str.[0-9]*]] = {{.*}}{memory:BLOCK_RAM}{sizeinfo:4}
+//CHECK-DEVICE: [[ANN7:@.str.[0-9]*]] = {{.*}}{memory:MLAB}{sizeinfo:4}
+//CHECK-DEVICE: [[ANN8:@.str.[0-9]*]] = {{.*}}{memory:DEFAULT}{sizeinfo:4}{bankwidth:8}
+//CHECK-DEVICE: [[ANN9:@.str.[0-9]*]] = {{.*}}{memory:DEFAULT}{sizeinfo:4}{max_private_copies:4}
+//CHECK-BOTH: [[ANN16:@.str[.0-9]*]] = {{.*}}foobar
```

Comment:
Is this label differ on host and device IRs?

Reply:
No, because the annotate attribute is not SYCL-specific.

Reply:
Then maybe to revert change in label's regexp?

Reply:
I misunderstood you at first -  it has no difference in the annotation itself, but the numbering differs (the first global string has `@str`, not `@str.[0-9]+`

Reply:
Ok? that is what I wanted to know.